RobotCode/src/main/java/frc/robot/constants/Ports.java

```diff
@@ -3,4 +3,8 @@
 public final class Ports
 {
 
+	public static final int ARM_LOWER_MOTOR = 0;
+	public static final int ARM_UPPER_MOTOR = 1;
+    public static final int ARM_GYRO = 2;
```

Comment:
No gyro on the arm. Read from TalonFX encoders. The gyro on the 2024 bot was a hacky solution to get around an issue we were having.